src/SyntaxError.php

```diff
@@ -0,0 +1,122 @@
+<?php
+namespace JakubOnderka\PhpParallelLint;
+
+class SyntaxError extends Error
```

Comment:
Do you know that `Error` is builtin class in php 7?

This leads to several issues:
1. if Error exists in this project it will be used. this is very bad practice to shadow classes. as if t he class is dropped, the root \Error will be used. bad for maintenance, etc
2. if it's written with the intent to extend \Error, you break older php support. or it's planned to load symfony polyfill?

Reply:
This feedback belongs with PR #44.